src/include/reservation.h

```diff
@@ -151,6 +153,23 @@ typedef struct pbsnode_list_ {
 	struct pbsnode_list_ *next;
 } pbsnode_list_t;
 
+/* Misc structures dealing with altering reservations */
+struct resv_alter_revert {
+	time_t rr_stime;
+	long rr_duration;
+	char *rr_select;
+};
+
+struct resv_alter {
+	time_t ra_stime;
+	time_t ra_etime;
+	long ra_duration;
+	char *ra_select;
+	long ra_state;
+	unsigned long ra_flags;
+	struct resv_alter_revert ra_revert;
```

Comment:
you added a lot of values, more than what was originally there, can you add some comments to explain the difference between `resv_alter.ra_stime` and `resv_alter_revert.rr_stime`

Reply:
The only new value is ra_revert.  There was a bug in existing code that if you modify a standing reservation multiple times, it only reverts back to the most recent change (i.e., A->B->C, it'd revert to B, not A).  My options were to fix the bug or exacerbate it by making it broken with select.  I chose to fix it.  I'll add the comment.